cep-20.2.md

```diff
@@ -0,0 +1,711 @@
+```
+
+## Context section
+
+```yaml
+# The context dictionary defines arbitrary key-value pairs for Jinja interpolation
+# and replaces the {% set ... %} commands commonly used in recipes
+context:
+  variable: test
+  # note that we can reference previous values, that means that they are rendered in order
+  other_variable: test_${{ variable }}
+```
+
+## Package section
+
+```yaml
+package:
+  # The name of the package
+  name: string
+  # The version of the package, following the conda version spec
+  # Note that versions are always strings (1.23 is _not_ a valid version since it is a `float`. Needs to be quoted.)
+  version: string
+```
+
+## Build section
+
+```yaml
+build:
+  # the build number
+  number: Option<integer>, defaults to 0
+  # the build string. This is usually ommited
+  string: Option<string>, defaults to a build string made from "package hash & build number"
+
+  # A list of conditions under which to skip the build of this package
+  skip: [list of expressions]
+
+  force_use_keys: [string]
+  force_ignore_keys: [string]
+
+  # wether the package is a noarch package, and if yes, wether it is "generic" or "python"
+  noarch: Option<"generic" | "python">
+
+  # PythonEntryPoint: `bsdiff4 = bsdiff4.cli:main_bsdiff4`
+  entry_points: [PythonEntryPoint]
+
+  # Use the python.app for the entrypoint (not exactly sure what that means in practice)
+  osx_is_app: bool (default false)
+
+  # used on conda-forge, still needed?
+  preserve_egg_dir: bool (default false)
+
+  # only used as a hack to down-prioritize packages
+  track_features: [string]
+
+  # skip compiling pyc for some files
+  skip_compile_pyc: [glob]
+
+  # do not soft- or hard-link these files, but always copy them
+  no_link: [glob]
+
+  # the script that is executed to build the package
+  # if it is only one element and ends with `.sh` or `.bat`
+  script: string | [string]
+
+  # linux only, list of rpaths
+  rpaths: [path] (defaults to ['lib/'])
+
+  # include files even if they are already in the environment as part of some other
+  # host dependency
+  always_include_files: [path]
+
+  # wether to relocate binaries or not. If this is a list of paths, then
+  # only the listed paths are relocated
+  binary_relocation: bool (defaults to true) | [glob]
+
+  # force file to be detected as TEXT file (for prefix replacement)
+  has_prefix_files: [path]
+
+  # force file to be detected as BINARY file (for prefix replacement)
+  binary_has_prefix_files: [path]
+
+  # ignore all or specific files for prefix replacement
+  ignore_prefix_files: bool | [path] (defaults to false)
+
+  # wether to detect binary files with prefix or not
+  detect_binary_files_with_prefix: bool (defaults to true on Unix and (always) false on Windows)
+
+  # Wether to include the recipe or not in the final package
+  include_recipe: bool (defaults to true)
+
+  # environment variables to either pass through to the script environment or set
+  script_env: [env_vars]
```

Comment:
I always felt this name was very ambiguous (specially due to the presence of `script` elsewhere). `env_vars` is a better term, or even `environment_variables`.

Reply:
I actually need to update the spec because we have already updated this in the recipe-format repository: https://github.com/prefix-dev/recipe-format/blob/f673bbc5778be4ce0d4e2765c89cf45948eb7caa/model.py#L130-L141

Although I just saw that we kept the `script_env` name. Happy to adjust that to something else...